addons/account/models/chart_template.py

```diff
@@ -255,6 +255,7 @@ def tax_template_changed(tax, template):
             return (
                 tax.amount_type != template.get('amount_type', 'percent')
                 or tax.amount != template.get('amount', 0)
+                or len(tax.repartition_line_ids) != len(values.get('repartition_line_ids'))
```

Comment:
```suggestion
                or len(tax.repartition_line_ids) != len(template.get('repartition_line_ids'))
```
Let's keep it consistent, it makes it clearer... don't you think?